.github/workflows/docker-pr-build.yml

```
@@ -155,9 +155,24 @@ jobs:
+    # Needed to push the image to the GitHub Container Registry (ghcr.io)
+    # and to comment on the pull request.
+    permissions:
+      contents: read
+      packages: write
+      pull-requests: write
     steps:
       - name: ⬇️ Checkout code
         uses: actions/checkout@v3
+      - name: 🏷️ Compute image tag from branch name
+        id: image_tag
+        run: |
```

actionlint (1.7.12)

[error] "github.head_ref" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/reference/security/secure-use#good-practices-for-mitigating-script-injection-attacks for more details

(expression)